frontend/src/components/article/parts/ArticleCard.tsx

```diff
@@ -10,16 +10,19 @@ type ArticleDetailsCardProps = {
 export default function ArticleCard(props:Readonly<ArticleDetailsCardProps>):React.ReactElement {
 
     return (
-        <div className={"articlesCard"}>
-            <article>{props.article.content}</article>
+        <div className={"articleCard"}>
+            <article >{props.article.content}</article>
             {props.article.npcIds.length > 0 &&
                 <DisplayTileGallery
                     urlPrefix={"/npc/"}
-                    tileData={props.article.npcIds.map((npcId:string) => ({id:npcId, name:"Jürgen"}))} /* For later: get the npc name, when there are npcs. */
+                    tileData={props.article.npcIds.map((npcId: string) => ({
+                        id: npcId,
+                        name: "Jürgen"
+                    }))} /* For later: get the npc name, when there are npcs. */
```

Comment:
You dont want comments in production code

Reply:
ok